sphinx_intl/transifex.py

```diff
@@ -14,6 +17,8 @@
 # ==================================
 # settings
 
+MINIMUM_VERSION = (1, 1, 0)
```

Comment:
How about `TRANSIFEX_CLIENT_MINIMUM_VERSION`?

Reply:
Changed to `TRANSIFEX_CLI_MINIMUM_VERSION`. I think they're using "client" on their python library and "cli" on their go library.